spinegeneric/cli/generate_figure.py

```diff
@@ -38,10 +38,13 @@
 
 # List subject to remove, associated with contrast
 SUBJECTS_TO_REMOVE = [
+    # CSA
     {'subject': 'sub-oxfordFmrib04', 'metric': 'csa_t1'},  # T1w scan is not aligned with other contrasts (subject repositioning)
     {'subject': 'sub-oxfordFmrib04', 'metric': 'csa_t2'},  # T1w scan is not aligned with other contrasts (subject repositioning)
+    {'subject': 'sub-mountSinai03', 'metric': 'csa_t2'},  # T2w was re-acquired (subject repositioning)
     # DTI
     {'subject': 'sub-beijingPrisma03', 'metric': 'dti_fa'},  # wrong FOV placement
+    {'subject': 'sub-sub-mountSinai03', 'metric': 'dti_fa'},  # T2w was re-acquired (hence wrong T2w -> DWI registration)
```

Comment:
should be `sub-mountSinai03` instead of `sub-sub-mountSinai03`

Reply:
thanks! fixed in 5c3811b